pandas/core/ops/docstrings.py

```diff
@@ -352,6 +356,12 @@ def _make_flex_doc(op_name, typ):
     if reverse_op is not None:
         _op_descriptions[reverse_op] = _op_descriptions[key].copy()
         _op_descriptions[reverse_op]["reverse"] = key
+        _op_descriptions[key][
+            "see_also_desc"
+        ] = f"Reversed {_op_descriptions[key]['desc']}"
```

Comment:
Do you mind writing a more descriptive description? I don't think for a user being shown `radd` for the first time, telling them `Reverse add` is very helpful. I wrote an example with a link in a previous comment that you can use as reference.